### PR TITLE
Merging Continuous Variables

### PR DESCRIPTION
**Pull request by shauryamalik**

### CODE REVIEW

**Comment by shauryamalik** [comment]
Updated again by removing excess code

**Comment by shauryamalik** [comment]
Sample EDA Separated

File: _posts/2020-09-03-Continuous Variables.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+```
+
+
+
+
+    (array([148., 723., 373., 135.,  51.,  19.,   4.,   3.,   2.,   2.]),
+     array([ 34900., 106910., 178920., 250930., 322940., 394950., 466960.,
+            538970., 610980., 682990., 755000.]),
+     <a list of 10 Patch objects>)
+
```
Comment (RTG8055) [inline _posts/2020-09-03-Continuous Variables.md]: remove all these?

File: _posts/2020-09-03-Continuous Variables.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+sns.lineplot(data=df[:100]['SalePrice'])
+```
+
+
+
+
+    <matplotlib.axes._subplots.AxesSubplot at 0x1910852fd30>
+
+
+
```
Comment (RTG8055) [inline _posts/2020-09-03-Continuous Variables.md]: this etc.

File: _posts/2020-09-03-Continuous Variables.md (hunk 1 above)
Comment (shauryamalik) [reply _posts/2020-09-03-Continuous Variables.md]: Done

File: _posts/2020-09-03-Continuous Variables.md (hunk 2 above)
Comment (shauryamalik) [reply _posts/2020-09-03-Continuous Variables.md]: Done

**Pull request merged**